### PR TITLE
Add moja global organization profile page

### PR DESCRIPTION
**Pull request by HarshCasper**
## Description

This PR:

- Adds moja global organization page
- Adds major projects being developed under moja global
- Adds minimal docs on how to get involved
- Adds contact information

### CODE REVIEW

**Review by shubhamkarande13 — APPROVED** [review]

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Welcome to moja global 👋
+
+![moja-global-banner](../assets/moja_global_banner.jpg)
+
+Moja global’s mission is to support ambitious climate action by developing pioneering, open-source software – including the groundbreaking FLINT software – to help users accurately and affordably estimate greenhouse gas emissions and removals from forestry, agriculture and other land uses (AFOLU).
```
Comment (chicken-biryani) [inline profile/README.md:5]: 'groundbreaking' may come across as too strong.

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+You can reach out to us through the following channels as well:
+
+- [info@moja.global](mailto:info@moja.global) : Ask questions about moja global relevant to the members and management with a point of focus for new user countries and policy makers.
+- [tsc@moja.global](mailto:tsc@moja.global) : Ask questions about contributing code, helping out with development, resolving issues and building new features with moja global teams.
+- [mentorship@moja.global](mailto:mentorship@moja.global) : Ask questions about participating in the community, sponsoring programs, funding opportunity and community development.
+- [Slack community](https://join.slack.com/t/mojaglobal/shared_invite/zt-o6ta1ug0-rVLjAo460~d7JbZ~HpFFtw ): Connect with other developers, scientists, policy makers and other interested people over our Slack community.
+
+We organize monthly Technical Steering Committee (TSC) meetings to discuss the future of moja global and its ecosystem. TSC meetings are open to all members of the community. We also conduct working group meetups every two weeks to discuss a specific area of interest.
```
Comment (chicken-biryani) [inline profile/README.md:53]: For the group meetings. Maybe the upcoming meeting's date, time, and link can be provided? So that it's open to all? Maybe?

**Review by chicken-biryani — APPROVED** [review]

File: profile/README.md (hunk 1 above)
Comment (shubhamkarande13) [reply profile/README.md:5]: I agree with Shloka!
I think it's better to stay humble. :)

**Review by aornugent — APPROVED** [review]

File: profile/README.md (hunk 2 above)
Comment (HarshCasper) [reply profile/README.md:53]: Hi @chicken-biryani 

Right now we don't have a public calendar. I will add this, once we have one. Thanks for the suggestion. I will make an issue for the same :+1:

File: profile/README.md (hunk 1 above)
Comment (HarshCasper) [reply profile/README.md:5]: Fixed! Thanks @chicken-biryani and @shubhamkarande13 :hugs:

**Pull request merged**